### PR TITLE
Add `max()`

### PR DESCRIPTION
**Pull request by Abdelrahman-Kh-Fouad**
#306 
i added  max only with i32 for three or two arguments

### CODE REVIEW

**Comment by Abdelrahman-Kh-Fouad** [comment]
i have changed extension of `tokenizer.re` by mistake i didn't see `build0.sh ` ,i will return it.

File: src/runtime/lpython_builtin.py [hunk 1]
```diff
@@ -212,7 +214,7 @@ def pow(x: i32, y: f64) -> f64:
 def pow(x: f64, y: i32) -> f64:
     return x**y
 
-
+#def max()
```
Comment (Thirumalai-Shaktivel) [inline src/runtime/lpython_builtin.py:217]:
```suggestion
```

File: tests/tests.toml [hunk 2]
```diff
@@ -229,3 +229,8 @@ tokens = true
 [[test]]
 filename = "tokens/indent1.py"
 tokens = true
+
+[[test]]
+filename ="../integration_tests/test_max.py"
```
Comment (Thirumalai-Shaktivel) [inline tests/tests.toml:234]: I think this file does not exist in the PR. Did you forget to include it?

File: src/lpython/semantics/python_comptime_eval.h [hunk 3]
```diff
@@ -162,6 +163,49 @@ struct PythonIntrinsicProcedures {
                 ASRUtils::type_to_str(arg_type) + "'", loc);
         }
     }
+    //max()
+    static ASR::expr_t * eval_max(Allocator &al , const Location &loc , Vec<ASR::expr_t*>&args){
+        LFORTRAN_ASSERT(ASRUtils::all_args_evaluated(args));
```
Comment (Thirumalai-Shaktivel) [inline src/lpython/semantics/python_comptime_eval.h:168]: `all_args_evaluated()` makes sure that you provide only the `Constant value`; so here the input for the `max` function would be Constant Integer, real, etc...

**Comment by Thirumalai-Shaktivel** [comment]
Resolve the conflicts, as suggested by Gagan (https://lfortran.zulipchat.com/#narrow/stream/311866-LPython/topic/GSoC.202022/near/278048279)

File: tests/tests.toml (hunk 2 above)
Comment (Abdelrahman-Kh-Fouad) [reply tests/tests.toml:234]: i included but i made conflict that remove things.

File: src/lpython/semantics/python_comptime_eval.h (hunk 3 above)
Comment (Abdelrahman-Kh-Fouad) [reply src/lpython/semantics/python_comptime_eval.h:168]: ok ,i got it

**Pull request closed without merging**